documentation/api/development.md

```diff
@@ -105,6 +74,52 @@ The API can be run in debug mode that allows for remote attach debugging (curren
 
 - You should now be able to hit set breakpoints throughout the API
 
+## Local (non-Docker)
+
+Run `export PY_RUN_APPROACH=local` to run API and test functions locally when running commands in the Makefile. For example, `make test` or `make format` will run outside of Docker.
+
+**Note:** even with the native mode, many components like the DB and API will only ever run in Docker, and you should always make sure that any implementations work within docker.
+
+Running in the native/local approach may require additional packages to be installed on your machine to get working.
+
+### Prerequisites
+
+1. Install the version of Python specified in [pyproject.toml](../../api/pyproject.toml)
+   [pyenv](https://github.com/pyenv/pyenv#installation) is one popular option for installing Python,
+   or [asdf](https://asdf-vm.com/).
+   - If using pyenv run `pyenv local <version>` to ensure that version will be used in subsequent steps
+2. Ensure that `python -V` and `python3 -V` are picking up that version.
+   - If not, run `pyenv init -` and/or restart your shell to ensure it was run automatically
+3. After installing and activating the right version of Python, install
+   [poetry](https://python-poetry.org/docs/#installation) and follow the instructions to add poetry to your path if necessary.
+
+   ```bash
+   curl -sSL https://install.python-poetry.org | python3 -
+   ```
+
+4. You'll also need [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+**Note:** All the following commands should be run from the `/api` directory.
+
+### Database setup: Run Migrations/Seeds
+
+If you haven't done local development before you'll need to execute the migrations and seed the DB with data using the steps in [database-local-usage.md](database/database-local-usage.md)
+
+### Services
+
+Individual services can be run through Docker, which can be useful in concert with non-Docker application development:
+
+* **OpenSearch**
+  * Run `make init-opensearch` setup the OpenSearch Container
+  * Run `make populate-search-opportunities` to push data previously seeded in the DB into the search index
+
+  If your DB or OpenSearch end up in an odd place, you can reset all the persistent storage using `make volume-recreate`.
+
+* **Localstack (local s3)**
+   * Run `make init-localstack`
+* **Mock OAuth server**
```

Comment:
this doesn't seem to be mentioned in any other documentation. Is it worth mentioning in this file what this is for and how / when to use it, maybe in the "User Authentication" section?